README.md

```diff
@@ -2,18 +2,63 @@
 
 This repository is reserved for Rancher Extensions created by Rancher Partners.  Extensions were introduced with Rancher 2.7.0. 
 
-The "Tested & Certified -- Rancher Extension" certification is part of the SUSE "Tested & Certified" product certification framework and it aims to address the growing need for a standardized, high-quality, and reliable ecosystem of Rancher Extensions that augment the capabilities of the Rancher by SUSE platform.
+### What are the requirements for adding a project to this repository?
 
-## Requirements
+TODO: Confirm with Troy - verified partner
```

Comment:
Do we need to run through the SUSE verified partner process for owners of extensions that we ship in this repo?